graphql.install

```diff
@@ -63,3 +63,12 @@ function graphql_update_8001(): void {
     $server->save();
   }
 }
+
+/**
+ * Preserve incorrect error reporting behaviour for PR #1161.
+ */
+function graphql_update_8201() :void {
+  \Drupal::configFactory()->getEditable('graphql.settings')
+    ->set('deprecated_swap_errors_and_extensions_in_results', TRUE)
+    ->save();
```

Comment:
Also tested this on dev site if the config schema does not exist, seems to always work, so should also be good.